docs/glossary.md

```diff
@@ -21,17 +27,16 @@ Collection, roughly speaking, is a set of data. A set of structured data is ofte
 
 ### Commit
 A commit is a point-in-time snapshot of the branch. It's a collection of object metadata and data, including paths and the object contents and metadata. Commits have their own commit metadata. Every repository has one initial commit with no parent commits. If a commit has more than one parent, it is a merge commit. lakeFS supports only merge commits with two parents.
-
 ### Cross-collection Consistency
 In lakeFS, a repository (and thus a branch) can span multiple collections. By providing branch, commit, merge and revert operations atomically on these branches, lakeFS achieves consistency guarantees across different logical collections. That is, data versioning is consistent across collections within a repository.
 
 <!---Learn more about cross-collection consistency here (link to CCC blog) -->
 
 ### Data Lifecycle Management
-In data-intensive applications, data should be managed through its entire lifecycle similar to how teams use to manage code. By doing so, we could leverage the best practices and tools from application lifecycle management (like CI/CD operations) and apply them to data. lakeFS offers data lifecycle management via [isolated data development environments](https://docs.lakefs.io/use_cases/iso_env.html) instead of shared buckets.
+In data-intensive applications, data should be managed through its entire lifecycle similar to how teams use to manage code. By doing so, we could leverage the best practices and tools from application lifecycle management (like CI/CD operations) and apply them to data. lakeFS offers data lifecycle management via [isolated data development environments]({{ site.baseurl }}/use_cases/iso_env.html) instead of shared buckets.
```

Comment:
Is there any reason not to use a standard relative URL?  Here is [an example](https://github.com/treeverse/lakeFS/blob/314518ea03c456c0c3dd353fdaf460b67104c2fc/docs/reference/authentication.md?plain=1#L100) of how we usually do it.  If there is no advantage, this style only adds complexity.

Also I prefer to link to `.md` files: that works exactly the same, but makes it easier to edit the files because it makes the name of the intended file clear.

Reply:
Updated the links to standard relative URLs.